ci/README.md

```diff
@@ -0,0 +1,110 @@
+# Continuous Integration Setup
+
+The CI environment is maintained by the SAP Business Technology Platform on Cloud Foundry team for Routing and Networking (CFN).
+
+All secrets used in the pipeline and surrounding tools are kept in the team's Vault. Those secrets are only necessary for [modifying the pipeline](#updating-or-modifying-the-pipeline).
+## Contact
+
+Credentials and admin access lie with that team. If you have questions or issues, please reach out via the [#haproxy-boshrelease](https://cloudfoundry.slack.com/archives/C0XRT9L22) channel on Slack.
+
+## Setup
+
+Concourse is used as CI system. There are two main types of tests and various release specific steps, all of which are defined in [pipeline.yml](pipeline.yml).
+
+* `unit-tests` runs a series of unit tests on the Ruby templating based configuration file generators and includes linters for all code and test code.
+* `acceptance-tests` and `acceptance-tests-pr` runs a series of acceptance tests developed in Go.
+  * `acceptance-tests-pr` is executed for each PR that is marked with the `run-ci` label, while
+  * `acceptance-tests` is run on new commits to `master`, e.g. after a PR has been merged.
+
+All tests run in Docker. The image `iacbox.common.repositories.cloud.sap/haproxy-boshrelease-testflight` is a built and cached version of building [`Dockerfile`](Dockerfile).
+
+***Note, August 2022***: The image used for acceptance tests is currently outdated and will be updated to a recent state in the near future. An updated version is so far only used for the autobump scripts.
```

Comment:
I'd spare this section if there is no issue with the outdated script.
Or create an issue if this is an issue

Reply:
Just saw the comment below, maybe cross-reference to the current limitation and spare one of these notes

Reply:
Those two are unrelated, i.e. the image used for the PR validation / acceptance tests being built some time ago but working fine vs. local tests not working in Docker for Mac (or likely new Docker engines in general).

I will change the wording of the note here. There is no problem with the state, just that the base image is a rolling release and for the auto bumping we needed to create a new version that also included an update to the latest Ubuntu user space.

Updating the image and having a single one that we can maintain regularly is definitely a short/mid-term goal that should be achievable reasonably easily.